public/style.css

```diff
@@ -65,4 +65,38 @@ input:checked + .slider:before {
 #floor2 {
```

Comment:
Why is there a CSS file in the root of the `public` folder, along with CSS files in `public/css`